en/embedding.html

```diff
@@ -273,10 +389,277 @@ <h2 id="examples">Examples</h2>
   Try the <a href="https://github.com/vespa-engine/sample-apps/tree/master/simple-semantic-search">
   simple-semantic-search</a> sample application. 
   The <a href="https://github.com/vespa-engine/sample-apps/tree/master/commerce-product-ranking">commerce-product-ranking</a> 
-  demonstrates using multiple embedders. Another complete example application using multiple embedders can be found in
-  <a href="https://github.com/vespa-engine/system-test/blob/master/tests/search/embedding/app_two_embedders/">this system test</a>.
+  demonstrates using multiple embedders. The 
+  <a href="https://github.com/vespa-engine/sample-apps/tree/master/multi-vector-indexing">multi-vector-indexing</a> sample-app demonstrates
+  how to use embedders with multiple inputs. 
 </p>
 
+<h2 id="model-config-reference">Model config reference</h2>
+<p>Embedder models uses the <a href="reference/config-files.html#model">model</a> type configuration. 
+  The <em>model</em> type configuration accepts attributes <code>model-id</code>, <code>url</code> or <code>path</code>, 
+  and multiple of these can be specified as a single config value, where one is used depending on the deployment environment:
+
+  If a model-id is specified and the application is deployed on Vespa Cloud, the <code>model-id</code> is used.
+  Otherwise, if a <code>url</code> is specified, it is used. Otherwise, <code>path</code> is used. </p>
```

Comment:
Better to use a list here, a la (rewrite with HTML)
```suggestion
  If a model-id is specified and the application is deployed on Vespa Cloud, the <code>model-id</code> is used.
  
  Otherwise, if a <code>url</code> is specified, it is used.
  
  Otherwise, <code>path</code> is used. </p>
```